### PR TITLE
[card-2] implement Expandable RichText

### PR DESCRIPTION
**Pull request by minhajdevcrew**
implement styling of custom tags present in the text string along with expand|collapse able text.

Closes [Card-2]

### CODE REVIEW

**Comment by minhajdevcrew** [comment]
[Expandable Rich Text](https://trello.com/c/4SYkc3nV/2-expandable-rich-text)

File: example/README.md
Comment (WaseemAbbas-DevCrew) [inline example/README.md:1]: update readme content

File: lib/expandable_rich_text.dart [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+    this.textDirection,
+    this.textAlign,
+    this.textScaleFactor,
+    this.maxLines = 2,
+    this.animation = false,
+    this.animationDuration,
+    this.animationCurve,
+    this.semanticsLabel,
+  }) : super(key: key);
+
```
Comment (WaseemAbbas-DevCrew) [inline lib/expandable_rich_text.dart:44]: Add documentation for all of the attributes so that the user is clear on how and when to use them.

**Comment by WaseemAbbas-DevCrew** [comment]
@minhajdevcrew please do the requested changes.

File: example/pubspec.yaml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+# versions available, run `flutter pub outdated`.
+dependencies:
+  flutter:
+    sdk: flutter
+
+
+  # The following adds the Cupertino Icons font to your application.
+  # Use with the CupertinoIcons class for iOS style icons.
+  cupertino_icons: ^1.0.2
+
```
Comment (WaseemAbbas-DevCrew) [inline example/pubspec.yaml:39]: Add a todo statement to clarify that we'll change the path once the package is live.

**Review by WaseemAbbas-DevCrew — CHANGES_REQUESTED** [review]

File: example/README.md
Comment (minhajdevcrew) [reply example/README.md:1]: will do in separate task.

**Review by WaseemAbbas-DevCrew — APPROVED** [review]

**Pull request merged**